README.md

```diff
@@ -0,0 +1,5 @@
+# Cambie Collective Node Server
+
+Simple Node/Express app which listens on port 3000 and returns 'Hello World!!!'
+
```

Comment:
I think we'll need `npm install` as well 🚀

Reply:
I agree. Thanks!

Reply:
Maybe you can use this :)

```suggestion
First run `npm install` to install the dependencies!
```